src/state/authentication/authentication.reducer.js

```diff
@@ -1,29 +1,28 @@
 import {
-  SIGNUP_ERROR,
-  LOGIN_ERROR,
-  SIGNUP_SUCCESS,
-  LOGIN_SUCCESS,
   CHANGE_PASSWORD_SUCCESS,
-  CHANGE_PASSWORD_ERROR,
   AUTH_SUCCESS,
   LOGOUT,
   DELETE_ACCOUNT_SUCCESS,
-  DELETE_ACCOUNT_ERROR,
   REMOVE_ERRORS,
   REMOVE_MESSAGES,
   CHECK_FIRST_LAUNCH,
+  AUTH_ERROR,
+  LOGIN_SIGNUP_SUCCESS
 } from './authentication.types';
 
+const initialState = {
+  token: '',
+  username: '',
+};
+
 const authReducer = (state, { type, payload }) => {
   switch (type) {
-    case SIGNUP_ERROR:
-    case LOGIN_ERROR:
+    case AUTH_ERROR:
       return {
         ...state,
         errorMessage: payload,
       };
-    case SIGNUP_SUCCESS:
-    case LOGIN_SUCCESS:
+    case LOGIN_SIGNUP_SUCCESS:
```

Comment:
can this also be just AUTH_SUCCESS like u combined the error types ?

Reply:
i tried but could not solve it, cuz in AUTH_SUCCESS
```
 case AUTH_SUCCESS:
      return {
        ...state,
        token: payload.token,
        username: payload.username,
      };
```
but in login signup 

```

    case LOGIN_SIGNUP_SUCCESS:
      return {
        ...state,
        token: payload.token,
        username: payload.user.username,
        errorMessage: '',
      };
```

sp i did not know how to pass  username: payload.user.username

Reply:
but actually i just can solve it in action creators hmmm i will try in other branch